ddtrace/debugging/_signal/metric_sample.py

```diff
@@ -78,3 +76,11 @@ def sample(self, _locals):
             self.meter.histogram(probe.name, value, probe.tags)
         elif probe.kind == MetricProbeKind.DISTRIBUTION:
             self.meter.distribution(probe.name, value, probe.tags)
+
+    @property
+    def message(self):
+        return ("Condition evaluation error for probe %s" % self.probe.probe_id) if self.errors else None
```

Comment:
Lets sync with agent team on how those messages should be formatted? I think all agents should generate the same messages as much as possible.